project_template/tasks.py

```diff
@@ -281,14 +281,38 @@ class TaskOwnedIncomeFromIndependentActivitiesTable(CountTableRowsTask):
     task_form = forms.TranscribeIndependentActivities
     storage_model = models.OwnedIncomeFromIndependentActivitiesTable
     # TODO - add child_class
-    child_class = None
+    child_class = None;
+
+
+class TaskOwnedIncomeFromGamblingRowEntry(DigitalizationTask):
+    task_form = forms.TranscribeOwnedIncomeFromGamblingRowEntry
+    template_name = "task/owned_income_from_gambling.html"
+
+    def save_verified_data(self, verified_data):
+        owner_person, created = models.Person.objects.get_or_create(
+            name=verified_data['owner_name'],
+            surname=verified_data['owner_surname'])
+
+        owned_income_from_gambling, created = models.OwnedIncomeFromGamblingTableEntry.objects.get_or_create(
+            holder_relationship=verified_data['holder_relationship'],
+            holder_name=verified_data['holder_name'],
```

Comment:
The `holder_name` and `holder_surname` form fields are supposed to populate the `owner_person` instance above. Please remove both of them from this model instance.